lib/services/trello/trello_checklist_resource.rb

```diff
@@ -34,9 +34,18 @@ def create_item(new_checklist_item)
 
   def update_item(card, updated_checklist_item)
     prepare_request
-    response = http_put trello_url(
-        "cards/#{card.id}/checklist/#{updated_checklist_item[:idChecklistCurrent]}/checkItem/#{updated_checklist_item[:idCheckItem]}"
-      ), updated_checklist_item.to_json
-    found_resource(response).merge(checklist_id: updated_checklist_item[:idChecklistCurrent])
+
+    response = found_resource(
+      http_put(
+        trello_url(
+          "cards/#{card.id}/checklist/#{updated_checklist_item[:idChecklistCurrent]}/checkItem/#{updated_checklist_item[:idCheckItem]}"
+        ),
+        updated_checklist_item.to_json
+      )
+    )
+
+    if response
+      response.merge(checklist_id: updated_checklist_item[:idChecklistCurrent])
+    end
```

Comment:
I guess this used to fail when trying to merge `:checklist_id` onto a missing response, so this method was either a successful response or an exception.

Do the callers of this method anticipate a nil response due to this change? It looks like some of the other services raise a RemoteError...

Reply:
Discussed and decided: a failure here can be ignored so we'll just return nil.